lib/screens/home/pages/profile/bloc/profile_page_bloc.dart

```diff
@@ -31,6 +41,21 @@ class ProfilePageBloc extends Bloc<ProfilePageEvent, ProfilePageState> {
         yield ProfilePageFailure(message: failure.message);
       }
     }
+  }
+
+  Stream<ProfilePageState> mapEventToRefreshRequested(ProfilePageEvent event) async* {
+    if (event is ProfilePageRefresh) {
+      yield ProfilePageLoading();
+      try {
+        _user = await userRepository.getCurrentUser();
+        yield ProfilePageSuccess(_user, message: event.message);
+      } on Failure catch (_) {
+        yield state;
```

Comment:
Add `Logger.root.error` or something like this

Reply:
Done